### ISSUE TITLE
Sodium/Rubidium caused Incense Candle to no longer render correctly [All loaders]

### ISSUE THREAD

**Issue by TelepathicGrunt** [TelepathicGrunt/Bumblezone#176]
![image](https://user-images.githubusercontent.com/40846040/205423042-d1d30316-c0d7-4b09-bff2-576316c8b0f1.png)

**Comment by TelepathicGrunt** [comment]
Issue is caused by Sodium's Biome Blend option. Turn that off as workaround for now.

![image](https://user-images.githubusercontent.com/40846040/205424907-7e6a6a23-c9ed-4c4e-b501-1415ac19b7e6.png)

**Comment by TelepathicGrunt** [comment]
https://github.com/CaffeineMC/sodium-fabric/issues/895

**Comment by TelepathicGrunt** [comment]
actual workaround is to use this mod:
https://www.curseforge.com/minecraft/mc-mods/unofficial-sodium-biome-blending-fix

**Comment by Pultex** [comment]
In 1.18 which I guess is not supported anymore

sodium with indium and ferrite core

make the sugar water not render as it states
Block file not found: the_bumblezone:sugar_water_block
this reverts it back to water [ guess as a fail safe from those mods]
crashes random then will just works fine

this also happens to botania so it is not just bumblezone

think the performance mods are not waiting till all things have loaded in
or is actively blocking things they think are dups or a waste of memory
like foam mod caused problems in the past

**Comment by TelepathicGrunt** [comment]
@Pultex Correct, I am not supporting 1.18.x or below. Though that error is talking about the block which is invisible and so, has no texture file for it. The fluid renders separately. Sugar Water looks just like a lighter version of water so no, the other mods aren't doing a fallback. That is sugar water itself you're seeing. I just tested 1.18.2 with the mods you specified as I was curious and I got no crash and can see the sugar water just fine.
![image](https://user-images.githubusercontent.com/40846040/208531630-5d3c5b21-c464-4df0-b2c0-afe0f0202525.png)
